app/frontend/src/service/__mocks__/index.ts

```diff
@@ -51,6 +52,22 @@ export const mockedService = ({
       ]),
     getGroupChatMessages: () => Promise.resolve(messages),
   },
+  requests: {
+    listHostRequests: () =>
+      Promise.resolve([
+        {
+          hostRequestId: 1,
+          fromUserId: 1,
+          toUserId: 2,
+          status: HostRequestStatus.HOST_REQUEST_STATUS_PENDING,
+          created: { seconds: Date.now() / 1000, nanos: 0 },
+          fromDate: "2020/12/01",
+          toDate: "2020/12/06",
+          lastSeenMessageId: 0,
+          latestMessage: messages[0],
+        },
+      ]),
+  },
```

Comment:
The Messages view in storybook says this mock doesn't exists so `hostRequests` is undefined. Not sure why?

Reply:
Yeah something weird is going on there, as `listHostRequests` seems to be overwritten by something that imports this mock... haven't really got much time to dig into this more yet. This is one of my gripes on using module state in anything as they make testing a pain, especially as we don't have jest here.

Reply:
Let's just leave it - it will be refactors when I take redux out completely